gov2/sqs/CreateLPQueue/CreateLPQueuev2_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+    "context"
+    "encoding/json"
+    "errors"
+    "fmt"
+    "io/ioutil"
+    "testing"
+    "time"
+
+    "github.com/aws/aws-sdk-go-v2/aws"
+    "github.com/aws/aws-sdk-go-v2/service/sqs"
+)
+
+type SQSCreateQueueImpl struct{}
+
+func (dt SQSCreateQueueImpl) CreateQueue(ctx context.Context,
+    params *sqs.CreateQueueInput,
+    optFns ...func(*sqs.Options)) (*sqs.CreateQueueOutput, error) {
+
+    /*
+        buckets := make([]*types.Bucket, 2)
+        buckets[0] = &types.Bucket{Name: aws.String("bucket1")}
+        buckets[1] = &types.Bucket{Name: aws.String("bucket2")}
+
+        output := &s3.ListBucketsOutput{
+            Buckets: buckets,
+        }
+    */
```

Comment:
Did you mean to remove this?

Reply:
Yes, done (and in CreateQueue/CreateQueuev2_test.go)